modules/ballerina-core/src/main/java/org/ballerinalang/bre/bvm/BLangVM.java

```diff
@@ -2617,7 +2617,11 @@ public void invokeAction(ActionInfo actionInfo, FunctionCallCPEntry funcCallCPEn
         ActionInfo newActionInfo = programFile.getPackageInfo(actualCon.getPackagePath())
                 .getConnectorInfo(actualCon.getName()).getActionInfo(actionInfo.getName());
 
-        invokeCallableUnit(newActionInfo, funcCallCPEntry);
```

Comment:
Do we need this ?

Reply:
This is already merged with separate PR, seems git doesn't reflect that.